tests/homework1/task1/vector_test.py

```diff
@@ -0,0 +1,74 @@
+import unittest
```

Comment:
If you take a look at your checks, you will notice that these tests did not launch, only matrix-related ones. This is because of a naming convention: just rename this file as `test_vector.py`, it should start with `test`.